### ISSUE TITLE
Feature Request - SplitOrMergeObjects for 3D

### ISSUE THREAD

**Issue by minh-doan** [CellProfiler/CellProfiler#3363]
Not sure if this was already discussed.

SplitOrMergeObjects in 3D analysis is demanded, as Median Filter and RemoveHoles sometimes result in over/under-segmented objects.

**Comment by derekthirstrup** [comment]
Yes, the lack of options for 3D nuclei declumping are causing issues for me with respect to the current 3D cell segmentation pipeline.
